python/pyarrow/_parquet.pyx

```diff
@@ -138,11 +138,11 @@ cdef class FileMetaData:
     property format_version:
 
         def __get__(self):
-            cdef int version = self.metadata.version()
-            if version == 2:
-                return '2.0'
-            elif version == 1:
+            cdef ParquetVersion version = self.metadata.version()
+            if version == ParquetVersion_V1:
                 return '1.0'
+            if version == ParquetVersion_V2:
+                return '2.0'
```

Comment:
This was an unrelated API change from PARQUET-828, but wanted to get the test suite passing